cmake/projects/theora/hunter.cmake

```diff
@@ -0,0 +1,20 @@
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    theora
+    VERSION
+    1.1.1-p0
+    URL
+    "https://github.com/drodin/theora/archive/hunter-1.1.1.zip"
```

Comment:
https://github.com/cpp-pm/theora - Please make sure to prefix the release with `v`. 
So:
here - 1.1.1-p0
release name in cpp-pm/theora - v1.1.1-p0
branch name in cpp-pm/theora - hunter-1.1.1

Reply:
done